skcosmo/feature_selection/_greedy.py

```diff
@@ -0,0 +1,154 @@
+"""
+Sequential feature selection
```

Comment:
Very stupid question here. We discussed it on the last dev meeting, but I don't understand, why do we concentrate on the feature selection? I understand, that we can simply pass `X.T` to the function for samples choosing, but it's not very elegant

Reply:
No stupid questions -- we're focusing on feature selection to start because the aim of feature selection is a bit clearer -- to determine a minimally-redundant, information-rich subset of features that can be used to minimize error in an ML task. 

One reason it's a bit more trivial because, as features represent the samples, the presence of redundancies carries less information, versus in sample selection redundancies encode the sample distribution. This gets at a core difficulty with sample selection -- there are many motivations for selecting samples -- from removing redundancies for classification problems (i.e. I want to winnow the samples to only those which define clusters) to representing a sample distribution to determining an active set for a sparse kernel, to name a few. Each of these motivations requires a different emphasis in the selection mechanism, and the successes are determined by different criteria. 

I hope that the long-windedness alone conveys a bit of the nuance needed with sample selection compared with feature selection.

Reply:
Ok, great, thank you!